viadot/flows/adls_to_azure_sql.py

```diff
@@ -74,6 +78,28 @@ def df_to_csv_task(df, path: str, sep: str = "\t"):
     df.to_csv(path, sep=sep, index=False)
 
 
+@task
+def check_column_order_task(
+    table: str = None,
+    df: DataFrame = None,
+    sqldb_credentials_secret: str = None,
+    if_exists: str = None,
```

Comment:
Let's keep the convention the same as in constructor parameters ✨ 

```suggestion
   if_exists: Literal["fail", "replace", "append", "delete"] = "replace",
```

However, add only those values that are being handled by the method.